src/handlers/assign/auto.ts

```diff
@@ -9,6 +10,8 @@ export const checkPullRequests = async () => {
   const logger = getLogger();
   const pulls = await getPullRequests(context);
 
+  if (ASSIGN_COMMAND_ENABLED) return;
```

Comment:
Execution should stop here if assign is enabled? Can you verify that's correct?

Reply:
Shouldn't we disable "Assign bounty hunter who opened linked draft pr" feature if assign is enabled?

Reply:
I don't think they are mutually exclusive, so we can have both enabled!

Reply:
Makes sense, so, you bet is it better to remove this line?

Reply:
I think so. 

I don't have an opportunity now to look at the entire file on my computer til much later today so it's hard to say with 100% confidence.